lib/controllers/widget/remote_commands.dart

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import 'package:flutter/material.dart';
+
+/// Pops an alert and on tap loads the function from the parameter.
+Future<int> deprovisionReason(BuildContext context) {
+  return showDialog(
+      context: context,
+      builder: (context) {
+        return DeprovisionReasonSelect();
+      });
+}
+
+/// Pop Up widget to show the possible reasons of a deprovision.
+class DeprovisionReasonSelect extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() => _DeprovisionReasonSelect();
+}
+
+class _DeprovisionReasonSelect extends State<DeprovisionReasonSelect> {
+  int _reason;
+  bool _option1 = false;
```

Comment:
instead of naming option1, name it deprovision_option

Reply:
I changed it to a enum.